enforcer/values.yaml

```diff
@@ -78,7 +78,9 @@ nameOverride:
 
 expressMode: false
 
-securityContext:
+securityContext: {}
```

Comment:
No need to change this. securityContext has a default values that required for enforcer to run. In case additional parameters have to be set it can be done by providing them thru the command line:
`helm upgrade --install --create-namespace --namespace aqua aqua-kube-enforcer aqua-helm/kube-enforcer --set global.gateway.address=<gateway_address>,global.gateway.port=443,certsSecret.autoGenerate=true,global.platform=k8s,global.enforcer.enabled=true,aquaSecret.kubeEnforcerToken=<kube_enforcer_token>,image.tag=2022.4,enforcer.enforcerToken=<enforcer_token>,enforcer.image.tag=2022.4,serviceAccount.create=true,clusterName=aks,global.imageCredentials.create=true,global.imageCredentials.repositoryUriPrefix=registry.aquasec.com,global.imageCredentials.registry=registry.aquasec.com,global.imageCredentials.username=<username>,global.imageCredentials.password=<password>,enforcer.expressMode=true,enforcer.securityContext.NewValue=blah-blah --dry-run | grep securityContext -B 5 -A 5`

```
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /home/ubuntu/.kube/config   
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /home/ubuntu/.kube/config  
      serviceAccount: aqua-kube-enforcer-agent-sa  
      containers:  
      - name: enforcer  
        image: "registry.aquasec.com/enforcer:2022.4"  
        imagePullPolicy: "Always"  
        securityContext:  
          NewValue: blah-blah  
          capabilities:  
            add:  
            - SYS_ADMIN  
            - NET_ADMIN
   ```